pkg/oauth/apis/oauth/validation/validation.go

```diff
@@ -81,6 +81,8 @@ func ValidateAccessTokenUpdate(newToken, oldToken *oauthapi.OAuthAccessToken) fi
 	allErrs := validation.ValidateObjectMetaUpdate(&newToken.ObjectMeta, &oldToken.ObjectMeta, field.NewPath("metadata"))
 	copied := *oldToken
 	copied.ObjectMeta = newToken.ObjectMeta
+	// allow only TimeoutsIn to be changed
+	copied.TimeoutsIn = newToken.TimeoutsIn
```

Comment:
what kind of validation ?

Reply:
Negative values?  TimeoutsIn > Expires does not make sense?  Some level of sanity checking would probably be nice.

Reply:
I guess we could do this, but checking TimeoutsIn !> ExpiresIn would cause some races with updates failing close to expiration time, so I am not sure I would want to fail them. After all a timeout greater than expire will cause no issues as token expiration always prevails.